src/pages/validator/external-chains/flow.mdx

```diff
@@ -154,14 +154,57 @@ EOF
 cat /etc/systemd/system/access-node.service
 sudo systemctl enable access-node
 ```
+
 ## Build gateway
 ```bash
 git clone https://github.com/onflow/flow-evm-gateway.git
+cd flow-evm-gateway
 go build -o evm-gateway cmd/main/main.go
 mv evm-gateway /usr/bin/
 ```
 
-### Gateway
+### EVM Gateway
+
+Each EVM Gateway operator needs to create an account on Flow as well as an account on EVM. The first one will be used to pay for fees on Flow, which means it needs to be sufficiently funded and monitored for balance (it shouldn’t fall to 0 without refunding it, as of now the node doesn’t monitor these, but will be added in the future, however, the funding won’t so it should be done manually). The second EVM account will be used to get refunded fees on EVM. That account shouldn’t have to be monitored for balance as the balance will always grow.
+```bash
+flow keys generate
+```
+It will generate an output like this
+```bash
+🔴️ Store private key safely and don't share with anyone!
+Private Key              5438b13e290b257f0c41e3b23fbf3b5f5f6d4e9b2d3797430d2d2d5484db9b37
+Public Key               9b85626369efe380ebd701f7189f2746fda26d09e1b63cb003bd84a5d33b662685c56427a0526a670efa88b0cb8da371cf9dc1de5ac6bbffb56e327a5c16e708
+Mnemonic                 sustain limb elbow awkward onion crouch truth trial until poverty famous feel
+Derivation Path          m/44'/539'/0'/0/0
+Signature Algorithm      ECDSA_P256
+```
+
+Visit https://faucet.flow.com, and use the generated Public Key, to create and fund your Flow account. Make sure to use the Flow address and the Private Key for the --coa-address & --coa-key flags.
+
+<tabs>
+    <tab-item title="Mainnet">
+        ```bash
+        ACCESS_NODE_GRPC_HOST="your_domian:9000"
+        FLOW_NETWORK_ID="flow-mainnet"
+        INIT_CADENCE_HEIGHT="211176670" # to be confirmed
```

Comment:
@StephenFluin please check with flow team on this

Reply:
This 211176670 height was for testnet, I assume for mainnet it might be different ?